airbyte-integrations/connectors/destination-langchain/unit_tests/docarray_indexer_test.py

```diff
@@ -0,0 +1,95 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+import unittest
+from unittest.mock import MagicMock, patch
+
+from airbyte_cdk.models import ConfiguredAirbyteCatalog
+from destination_langchain.config import DocArrayHnswSearchIndexingModel
+from destination_langchain.indexer import DocArrayHnswSearchIndexer
+from langchain.document_loaders.base import Document
+
+
+class BaseIntegrationTest(unittest.TestCase):
```

Comment:
Should it be in "integration_tests` if it's not a unit test?

Reply:
It's a unit-test I copied pasted the name - fixed